src/labbook/components/thumbnail-chooser/thumbnail-chooser.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "@testing-library/jest-dom";
```

Comment:
Since this is a side-effect import (i.e. you don't actually import any identifiers that are used by this test file), it is conventional to put this import in `setupTests.js|ts` so that it's available to all tests.